src/test/java/com/endava/internship/mocking/service/InMemPaymentRepositoryTest.java

```diff
@@ -0,0 +1,99 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.repository.InMemPaymentRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.NoSuchElementException;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class InMemPaymentRepositoryTest {
+
+    InMemPaymentRepository paymentRepository;
+
+    Payment payment = new Payment(1,20.0,"Payment 1");
```

Comment:
why not adding initialization in BeforeEach?

Reply:
Done